juju/juju
# Cloud commands operate on controller by default

## Description of change

The various cloud commands used to add/remove/list/update clouds are changed to operate by default on the controller rather than the local cache. After bootstrap, the most common use case is to manage clouds on the running controller, so the default command behaviour is now adjusted to cater for that.

If a new cloud is needed to be added to the local cache so another controller can be bootstrapped, there's a new --local option for that.

## QA steps

Run the various cloud commands with/without --local and check that the controller or local cache is updated as specified.
